is_core/forms/fields.py

```diff
@@ -29,6 +32,9 @@ def has_changed(self, initial, data):
     def validate(self, value):
         raise ValidationError(ugettext('Readonly field can not be validated'))
 
+    def get_bound_field(self, form, field_name):
+        return ReadonlyBoundField(form, self, field_name)
```

Comment:
novinka v 1.8 djangu jde to definovat zde a nemusím tak přepisovat formulář (kde se kvůli tomu odstraňovalo cachování form field)